src/lib.rs

```diff
@@ -70,7 +70,7 @@
 //!
 //! Other crate features can also be useful beyond the version support:
 //!
-//! * `macros` - adds the `uuid!` macro that can parse UUIDs from string literals at compile time.
+//! * `macros` - enhances the diagnostics of `uuid!` macro.
```

Comment:
```suggestion
//! * `macro-diagnostics` - enhances the diagnostics of `uuid!` macro.
```

Reply:
It's a miss when I rebased the commits :joy: